### ISSUE TITLE
How to create an issue?

### ISSUE THREAD

**Issue by brasilikum** [xanzy/go-gitlab#33]
```go
project, _, err := git.Projects.GetProject("georg/test-timetracking")
Expect(err).NotTo(HaveOccurred())
Expect(*(project.Name)).To(Equal("test-timetracking"))
_, _, err = git.Issues.CreateIssue(
	"georg/test-timetracking",
	&gitlab.CreateIssueOptions{
		Title:       "Hello World",
		Description: "I am the Description",
		AssigneeID:  2,
	},
)
Expect(err).NotTo(HaveOccurred())
```
Why does this result in an error 500? How do I create a new Issue then?

**Comment by svanharmelen** [comment]
Not sure, I use this is my functional tests which work as expected and looks just about the same as your example:

```go
opt1 := &gitlab.CreateIssuesOptions{
  Title:  "Auto test issue",
  Labels: []string{"bug", "testlabel2"},
}

_, _, err := git.Issues.CreateIssue("svanharmelen/test", opt1)
if err != nil {
  log.Fatal(err)
}
```

Maybe you should first check with `curl` if you are able to make this call directly. As maybe it's not related to this package, but to the values you supply or your Gitlab server? For some details/pointers on how to use `curl` for this, see the docs: http://doc.gitlab.com/ce/api/issues.html#new-issue

**Comment by brasilikum** [comment]
May be related to this issue, where users are not able to get the project by namespace/projectname https://github.com/gitlabhq/gitlabhq/issues/4921

**Comment by svanharmelen** [comment]
This should be fixed by PR #48
